ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -4647,7 +4647,7 @@
           "APP-SOL-UNSPEC-PROFILE",
           "APP-SOL-SPEC-PROFILE"
         ],
-        "CRUD": "RU"
+        "CRUD": "CRU"
```

Comment:
why would we change permission to this? isn't it the field to generate response document for respondent (i.e. defendant?)

Reply:
Needed that permission to avoid this document creation error:
ERROR [http-nio-4452-exec-9] u.g.h.c.d.service.common.AccessControlServiceImpl User doesn't have required access for caseField=respondent1GeneratedResponseDocument

Reply:
Changed permissions for just the required roles.